packages/vats/src/virtual-purse.js

```diff
@@ -9,9 +8,9 @@ import '@agoric/notifier/exported.js';
 
 /**
  * @template T
- * @typedef {import('@agoric/eventual-send').EOnly<T>} EOnly An object roughly
- * of type T that is only allowed to be consumed via eventual-send.  This allows
- * the object to return Promises wherever it wants to, even when T demands a
+ * @typedef {import('@agoric/far').EOnly<T>} EOnly An object roughly of type T
+ * that is only allowed to be consumed via eventual-send.  This allows the
+ * object to return Promises wherever it wants to, even when T demands a
```

Comment:
any particular reason to duplicate this documentation here?

Reply:
Not really.  I've removed this comment in favour of just a bare `@typedef`.